### ISSUE TITLE
Successful login does not continue past /signin

### ISSUE THREAD

**Issue by davidjmeier** [subspacecloud/subspace#23]
I recently updated to the latest build and realized the other day I can no longer login.  Successful auth just goes back to the "Admin Sign In".  Failed auth shows "Sign in failed. Please try again".  Are there auth logs to check in the running container? 

I'm creating the container as follows:

```
docker create \
    --name wireguard-data \
    --restart always \
    --network host \
    --cap-add NET_ADMIN \
    --volume /usr/bin/wg:/usr/bin/wg \
    --volume /data:/data \
    --env SUBSPACE_HTTP_HOST=wg.myfakedomain.com \
    --env SUBSPACE_LETSENCRYPT=false \
    --env SUBSPACE_HTTP_INSECURE=true \
    subspacecloud/subspace:latest
```

**Comment by nnadams** [comment]
This happened to me, too, and I just had to correct the SUBSPACE_HTTP_HOST value to be the exact IP/domain name I was using in the browser.

**Comment by davidjmeier** [comment]
You are correct sir.  Thank you.  I had two DNS entries in my local resolver.  So... For example, with the top hostnames as higher precedence:
- localnetname.lab.local
- wg.externaldomain.com

I just realized that if I did reverse resolution locally on my network I'd get the localnetname.lab.local, but SUBSPACE_HTTP_HOST was set to wg.externaldomain.com.  Changed the order of precedence and everything works as normal.  I don't remember changing the order of DNS, so I didn't even think to look at that.  Thanks, again for the tip!
